ansible/TestbedProcessing.py

```diff
@@ -213,11 +213,14 @@ def makeTestbed(data, outfile):
                 server = groupDetails.get("server")
                 vm_base = groupDetails.get("vm_base")
                 dut = groupDetails.get("dut")
+                ptf = ""
                 comment = groupDetails.get("comment")
 
                 # catch empty types
                 if not groupName:
                     groupName = ""
+                else:
+                    ptf = "ptf_" + groupName
```

Comment:
according to this file https://github.com/Azure/sonic-mgmt/blob/master/tests/vtestbed.csv
ptf entry is not "ptf_" + groupName.
As I remember "ptf" column has to be a host name defined in inventory